### ISSUE TITLE
[Programmatic Access – Make Code – Project Setting] Incorrect role and state are defined for toggle button controls present on 'Project Setting' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3683]
**User Experience:**
Users who depends upon screen reader get confused if the Screen reader reads incorrect role and state of any control. They'll not be able to know whether it is a link or a button or whether it is a non interactive control. Users will understand it to be a interactive control as screen reader announces link while navigating in scan mode and hence it will not be a good usable experience for the users. 

**Note**: User credentials should NOT be included in the bug.

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Project Settings" option and select it.
7. Navigate though all the controls appearing after select "Project Setting" option i.e. No Pairing Required: Anyone can connect via Bluetooth, Passkey pairing: Pairing requires 6 digit key to pair etc. 
8. Verify that Incorrect role and state are defined for toggle button controls.

**Actual Result:**
Incorrect role and state are defined for toggle button controls present on 'Project Setting' screen. Screen reader is reading as 'No Pairing Required: Anyone can connect via Bluetooth checkbox unchecked, JustWorks pairing (default): Pairing is automatic once the pairing is initiated checkbox checked & Passkey pairing: Pairing requires 6 digit key to pair checkbox unchecked'.

**Expected Result:**
Correct role and state should be define for toggle button controls present on 'Project Setting' screen. Screen reader should be read as 'No Pairing Required: Anyone can connect via Bluetooth toggle button off, JustWorks pairing (default): Pairing is automatic once the pairing is initiated toggle button on & Passkey pairing: Pairing requires 6 digit key to pair toggle button off.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 4.1.2 – Name, Role, Value ](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG4.1.2; #Win10-Edge(Chromium)

![WCAG4 1 2_Project Setting_Incorrect role and state is defined](https://user-images.githubusercontent.com/72871089/99029810-79494100-2599-11eb-8a5c-89168c5e4044.jpg)

**Comment by jwunderl** [comment]
These are checkboxes visually styled as toggle buttons; reading them out as checkboxes makes sense for a screen reader where they will not receive the visual hint of the toggles.
